proposals/2020/NMEC Implementation.md

```diff
@@ -0,0 +1,193 @@
+- The concept of an `auc:DerivedModel` is introduced.  It represents the generic concept of either a baseline or reporting model (referred to as an adjustment model in [3]).
+- The `auc:DerivedModel` complex type will be able to be instantiated as a child of the following elements:
+    - `auc:Scenario/auc:ScenarioType/auc:CurrentBuilding`
+        - It is expected that models for baseline periods would be embedded here (regardless if modeled or measured)
+        - `auc:DerivedModelPerformance` and child elements should be captured here
+        - All elements starting with `auc:BaselinePeriod` should be captured here
+    - `auc:Scenario/auc:ScenarioType/auc:PackageOfMeasures`
+        - It is expected that models for reporting periods would be embedded here (regardless if modeled or measured)
+        - All elements starting with `auc:ReportingPeriod` should be captured here
+- Multiple `auc:DerivdeModels` should be able to be captured, as users may want to test different `auc:ModelType`s
+- The `auc:DerivedModel` will be a complex type that would look something like:
+```xml
+  <auc:DerivedModels>
+    <auc:DerivedModel ID="Model1">
+      <auc:DerivedModelPeriod>[Baseline, Reporting]</auc:DerivedModelPeriod>
+      <auc:DerivedModelBaselineID/>
+      <auc:DerivedModelInputs>
+        <auc:ModelType>[2P SLR, 3P heating CPM, 3P cooling CPM, 4P CPM, 5P CPM]</auc:ModelType>
+        <auc:NormalizationMethod>[Forecast, Backcast, Standard Conditions]</auc:NormalizationMethod>
+        <auc:BaselinePeriodStartDate>2016-12-01</auc:BaselinePeriodStartDate>
+        <auc:BaselinePeriodEndDate>2017-12-01</auc:BaselinePeriodEndDate>
+        <auc:ReportingPeriodStartDate/>
+        <auc:ReportingPeriodEndDate/>
+        <auc:IntervalFrequency>Monthly</auc:IntervalFrequency>
+        <auc:ResponseVariable>
+          <auc:EnergyResource>Electricity</auc:EnergyResource>
+          <auc:EnergyResourceUnits>kWh</auc:EnergyResourceUnits>
+          <auc:EndUse>All end uses</auc:EndUse>
+        </auc:ResponseVariable>
+        <auc:ExplanatoryVariables>
+          <auc:ExplanatoryVariable>
+            <auc:ExplanatoryVariableName>[Temperature, Relative Humidity, GHI, DHI, DNI, Time-of-week, Time-of-day,...]</auc:ExplanatoryVariableName>
+            <auc:ExplanatoryVariableUnits>[Fahrenheit, Percent, W/m2, Hour of week, Hour of day, ...]</auc:ExplanatoryVariableUnits>
+            <auc:StandardConditionDataSource>[TMY3, ...]</auc:StandardConditionDataSource>
+          </auc:ExplanatoryVariable>
+        </auc:ExplanatoryVariables>
+      </auc:DerivedModelInputs>
+      <auc:DerivedModelParameters>
+        <auc:BetaCooling>1.11</auc:BetaCooling>
```

Comment:
I'm wondering if this should be more generic such as:

```
<DerivedModelParameters>
    <Parameters>
       <Parameter>
            <name>BetaCooling</name>
            <value>1.11</value>
       </Parameter>
       <Parameter>
           <name>BetaHeating</name>
           <value>2.51</value>
       </Parameter>
   </Parameters>
</Dervived...>

Reply:
I'm planning on leaving as is for now and focusing on supporting specific model types (G14 and TOWT) identified as industry standard.  Can look to add more support down the line and with feedback (maybe get feedback from ASHRAE?)